tools/flaky-test-reporter/ghutil/ghutil.go

```diff
@@ -0,0 +1,327 @@
+	issueListOptions := github.IssueListByRepoOptions{
+		State:  string(IssueAllState),
+	}
+	if len(labels) > 0 {
+		issueListOptions.Labels = labels
+	}
+	
+	var res []*github.Issue
+	options := &github.ListOptions{}
+	genericList, err := depaginate(
+		fmt.Sprintf("listing genericList with label '%v'", labels),
+		maxRetryCount,
+		options,
+		func() ([]interface{}, *github.Response, error) {
+			page, resp, err := client.Issues.ListByRepo(ctx, org, repo, &issueListOptions)
+			var interfaceList []interface{}
+			if nil == err {
+				for _, issue := range page {
+					interfaceList = append(interfaceList, issue)
+				}
+			}
+			return interfaceList, resp, err
+		},
+	)
+	for _, issue := range genericList {
+		res = append(res, issue.(*github.Issue))
+	}
+	return res, err
+}
+
+// CreateIssue creates issue
+func CreateIssue(org, repo, title, body string) (*github.Issue, error) {
+	issue := &github.IssueRequest{
+		Title: &title,
+		Body:  &body,
+	}
+
+	var res *github.Issue
+	_, err := retry(
+		fmt.Sprintf("creating issue '%s %s' '%s'", org, repo, title),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			res, resp, err = client.Issues.Create(ctx, org, repo, issue)
+			return resp, err
+		},
+	)
+	return res, err
+}
+
+// CloseIssue closes issue
+func CloseIssue(org, repo string, number int) error {
+	return updateIssueState(org, repo, IssueCloseState, number)
+}
+
+// ReopenIssue reopen issue
+func ReopenIssue(org, repo string, number int) error {
+	return updateIssueState(org, repo, IssueOpenState, number)
+}
+
+// ListComments gets all comments from issue
+func ListComments(org, repo string, number int) ([]*github.IssueComment, error) {
+	var res []*github.IssueComment
+	options := &github.ListOptions{}
+	genericList, err := depaginate(
+		fmt.Sprintf("commenting issue '%s %s %d'", org, repo, number),
+		maxRetryCount,
+		options,
+		func() ([]interface{}, *github.Response, error) {
+			page, resp, err := client.Issues.ListComments(ctx, org, repo, number, nil)
+			var interfaceList []interface{}
+			if nil == err {
+				for _, issue := range page {
+					interfaceList = append(interfaceList, issue)
+				}
+			}
+			return interfaceList, resp, err
+		},
+	)
+	for _, issue := range genericList {
+		res = append(res, issue.(*github.IssueComment))
+	}
+	return res, err
+}
+
+// GetComment gets comment by comment ID
+func GetComment(org, repo string, commentID int64) (*github.IssueComment, error) {
+	var res *github.IssueComment
+	_, err := retry(
+		fmt.Sprintf("getting comment '%s %s %d'", org, repo, commentID),
+		maxRetryCount,
+		func() (*github.Response, error) {
+			var resp *github.Response
+			var err error
+			res, resp, err = client.Issues.GetComment(ctx, org, repo, commentID)
+			return resp, err
+		},
+	)
+	return res, err
+}
+
+// CreateComment adds comment to issue
+func CreateComment(org, repo string, number int, commentStr string) error {
```

Comment:
s/number/issueNumber

I also think that just "comments" is cleaner and offers better readability.

Reply:
done. renamed to "commentBody" to make it different from "comment" struct